gravity.sh

```diff
@@ -43,7 +45,7 @@ else
 fi
 
 #Remove the /* from the end of the IPv4addr.
-IPv4addr=${IPv4_address%/*}
+IPv4_address=${IPv4_address%/*}
```

Comment:
[Linux-File-System-Hierarchy-blackMORE-Ops.pdf](https://github.com/pi-hole/pi-hole/files/546177/Linux-File-System-Hierarchy-blackMORE-Ops.pdf)

This is the guide that we try to go by with the location of files, just something for the insomnia reading shelf. :)